roles/openshift_manageiq/vars/main.yml

```diff
@@ -9,6 +9,20 @@ manageiq_cluster_role:
       verbs:
       - '*'
 
+manageiq_metrics_admin_clusterrole:
```

Comment:
@mwringe @detiber not sure if this belongs here... also how do you remember to keep this in sync these with `origin-metrics`?